mkdocs.yml

```diff
@@ -56,10 +56,17 @@ markdown_extensions:
       emoji_generator: !!python/name:materialx.emoji.to_svg
   - pymdownx.inlinehilite
   - pymdownx.snippets
-  - pymdownx.superfences
+  - pymdownx.superfences:
+      custom_fences:
+        - name: mermaid
+          class: mermaid
+          format: !!python/name:pymdownx.superfences.fence_div_format
   - tables
   - attr_list
   - md_in_html
   - meta
   - admonition
   - pymdownx.details
+
+extra_javascript:
+  - https://unpkg.com/mermaid@10.9.0/dist/mermaid.min.js
```

Comment:
My bad I missed this PR when reviewing #568.

But as I was mentioning in my [comment](https://github.com/scality/runner-manager/pull/568#discussion_r1550072227), I believe I was able to get those mermaid graph rendered without the need of specifying those extra settings.

```suggestion
```

Give it another try and let me know if it's working for you.